deployment/build_conda_packages.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+ 
+# This script creates conda packages for moldesign.
+#
+# The latest moldesign source is obtained from the source for this repository.
+#
+# The moldesign version is obtained using the latest git tag.
+#
+# ALl conda downloads and builds are created in directories under $HOME.
+#
+cd conda
+
+# Install minconda for Python 3.6
```

Comment:
Question: is this part necessary if we've already installed miniconda in the environment?

Reply:
If miniconda is there then we don't need to do this. Just need to make sure that conda-build and anaconda-client are installed.